extensions/requires-scopes/.gitignore

```diff
@@ -0,0 +1,3 @@
+target
+build
+.build.lock
```

Comment:
Maybe we could have a .gitignore for the whole repo, since the structure is going to be very similar between extensions.

Reply:
Yeah, I'll add that in the next PR